base_tier_validation_waiting/tests/test_tier_validation.py

```diff
@@ -0,0 +1,101 @@
+# Copyright (c) 2022 brain-tec AG (https://braintec.com)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo_test_helper import FakeModelLoader
+
+from odoo.tests.common import SavepointCase, tagged
+
+
+@tagged("post_install", "-at_install")
+class TierTierValidation(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TierTierValidation, cls).setUpClass()
+
```

Comment:
can you set tracking_disable + no_reset_password context keys pls?

Reply:
I meant class wise ;)
```suggestion
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```